core/src/main/java/com/opensymphony/xwork2/validator/AbstractActionValidatorManager.java

```diff
@@ -1,289 +0,0 @@
-                LOG.debug("Running validator: {} for object {} and method {}", validator, object, method);
-
-                FieldValidator fValidator = null;
-                String fullFieldName = null;
-
-                if (validator instanceof FieldValidator) {
-                    fValidator = (FieldValidator) validator;
-                    fullFieldName = fValidator.getValidatorContext().getFullFieldName(fValidator.getFieldName());
-
-                    if ((shortcircuitedFields != null) && shortcircuitedFields.contains(fullFieldName)) {
-                        LOG.debug("Short-circuited, skipping");
-                        continue;
-                    }
-                }
-
-                if (validator instanceof ShortCircuitableValidator && ((ShortCircuitableValidator) validator).isShortCircuit()) {
-                    // get number of existing errors
-                    List<String> errs = null;
-
-                    if (fValidator != null) {
-                        if (validatorContext.hasFieldErrors()) {
-                            Collection<String> fieldErrors = validatorContext.getFieldErrors().get(fullFieldName);
-
-                            if (fieldErrors != null) {
-                                errs = new ArrayList<>(fieldErrors);
-                            }
-                        }
-                    } else if (validatorContext.hasActionErrors()) {
-                        Collection<String> actionErrors = validatorContext.getActionErrors();
-
-                        if (actionErrors != null) {
-                            errs = new ArrayList<>(actionErrors);
-                        }
-                    }
-
-                    validator.validate(object);
-
-                    if (fValidator != null) {
-                        if (validatorContext.hasFieldErrors()) {
-                            Collection<String> errCol = validatorContext.getFieldErrors().get(fullFieldName);
-
-                            if ((errCol != null) && !errCol.equals(errs)) {
-                                LOG.debug("Short-circuiting on field validation");
-
-                                if (shortcircuitedFields == null) {
-                                    shortcircuitedFields = new TreeSet<>();
-                                }
-
-                                shortcircuitedFields.add(fullFieldName);
-                            }
-                        }
-                    } else if (validatorContext.hasActionErrors()) {
-                        Collection<String> errCol = validatorContext.getActionErrors();
-
-                        if ((errCol != null) && !errCol.equals(errs)) {
-                            LOG.debug("Short-circuiting");
-                            break;
-                        }
-                    }
-
-                    continue;
-                }
-
-                validator.validate(object);
-            } finally {
-                validator.setValidatorContext(null);
```

Comment:
As far as I can tell this doesn't do anything. Probably a remnant from when we used to cache the Validators instead of the ValidatorConfigs.